google/cloud/pubsub/version.h

```diff
@@ -18,32 +18,6 @@
 #include "google/cloud/pubsub/version_info.h"
 #include "google/cloud/version.h"
 
-#define GOOGLE_CLOUD_CPP_PUBSUB_NS                                     \
-  GOOGLE_CLOUD_CPP_VEVAL(GOOGLE_CLOUD_CPP_PUBSUB_CLIENT_VERSION_MAJOR, \
-                         GOOGLE_CLOUD_CPP_PUBSUB_CLIENT_VERSION_MINOR)
-
-namespace google {
-/**
```

Comment:
I can see why you would remove these, but they show up in the documentation for Pub/Sub:

https://googleapis.dev/cpp/google-cloud-pubsub/latest/namespaces.html

If you remove them from here, the documentation will be gone.

Reply:
Oh, I didn't know that was used for anything. Added them back.

Reply:
It is possible they are redundant, but let's separate that cleanup if that is the case.